internal/service/ec2/ebs_snapshot.go

```diff
@@ -73,6 +80,23 @@ func ResourceEBSSnapshot() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"storage_tier": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.Any(
+					validation.StringInSlice(ec2.TargetStorageTier_Values(), false),
+					validation.StringInSlice([]string{"standard"}, false), //Enum slice does not include `standard` type.
+				),
+			},
+			"permenent_restore": {
```

Comment:
Here and elsewhere, should be _perm_**a**_nent_restore_

Reply:
![](https://e7.pngegg.com/pngimages/351/544/png-clipart-the-simpsons-tapped-out-homer-simpson-bart-simpson-moe-szyslak-d-oh-simpsons-mammal-heroes.png)

Reply:
fixed